src/Components/Pronunciation/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { notification } from "antd";
+import { AiTwotoneSound } from "react-icons/ai";
```

Comment:
Remove `react-icons`. Since we're using Ant Design, let's be consistent and use its [icon pack](https://ant.design/components/icon/) instead. You can use `<SoundOutlined />`.